keyboards/levinson/keymaps/jyh/rules.mk

```diff
@@ -0,0 +1,6 @@
+ifndef QUANTUM_DIR
+	include ../../../../Makefile
+endif
+
+BACKLIGHT_ENABLE=yes
+USE_I2C=yes
```

Comment:
This doesn't actually do anything, and you should define this in the config.h file instead.

```suggestion
```